src/QMCApp/qmcapp.cpp

```diff
@@ -141,18 +138,20 @@ int main(int argc, char** argv)
       ++i;
     }
     int in_files = fgroup1.size();
-    std::vector<std::string> inputs(in_files * clones + fgroup2.size());
+    std::vector<std::string> inputs(in_files + fgroup2.size());
```

Comment:
could you replace `in_files` with `fgroup1`? I would also recommend renaming `fgroup1` as `f_names_cmd` and `fgroup2` as `f_names_txt` to help reading the code.

Reply:
Done